### DISCUSSION TITLE
Muse Guitars Vol. 2 — Mandolin (note volume and pitch)

### DISCUSSION THREAD

**Discussion by scorster** [musescore/MuseScore#25963] [General]
The mandolin included in Muse Guitars Vol. 2 is quite impressive! **Aside from a small array of glitches, the Mandolin exemplifies the sonic quality I've hoped to hear in MuseSounds and MuseScore Studio.**  

Regarding the various playback glitches, it's unclear if the Mandolin's undesirable artifacts are:

     • due to issues within the MuseSounds Guitars Vol 2 library
     • due to MuseScore Studio's audio engine
     • or due to the way MuseSounds and MuseScore Studio's audio engine interact

So I thought it reasonable to open a discussion here.

I've posted some audio examples, Pros / Cons, and discussion at the following .org links:

     https://musescore.org/en/node/372335#comment-1273411

     https://musescore.org/en/node/372335#comment-1273590

And here's a link to a very simple example score highlighting note playback volume at .com:

     https://musescore.com/user/35880724/scores/22643767

Here's another .org post on the topic of note playback volume:

     https://musescore.org/en/node/372335#comment-1278164

scorster

**Comment by scorster** [comment]
## Bug report sent to MuseHub

Today I sent a bug report to MuseHub. It includes a link to _this_ discussion and my recommendation that MuseHub comment here. 

Hopefully someone from MuseHub or MuseSounds will indeed respond here.

## Here's the text of the bug report

The only instrument I like in **MuseSounds Guitars Vol. 2** is its Mandolin. Unfortunately it's frequently problematic. 

My system is: 
     • MacOS 13.5.2
     • MacBook Air M2 24GB 
     • MuseScore 4.4.4 / 4.4.5
     • MuseSampler 0.100.10
     • MuseHub 2.0.25    (**2025-03-21**: I upgraded to **MuseHub 2.1.3** at the MS4.5.1 release)
     • my MacOS system library shows a number of MuseSounds downloaded, including
      ~/Library/Application Support/Muse Hub/Downloads/Instruments/Muse Guitars Vol 2  << Dated: 2024-12-29

Please see this Github discussion for an overview:

     https://github.com/orgs/musescore/discussions/25963

Here's a report on .org regarding the Vol. 2 Mandolin:

     https://musescore.org/en/node/372335#comment-1273411

This link opens a demonstration score on .com that illustrates various glitches: 

     https://musescore.com/user/35880724/scores/22643767

And at this link on Audio.com you can hear pitch issues, like the second accompaniment chord which is an F. Also you'll hear other notes struggling to start. And playback turns into a jagged mess when the tempo is above 130 BPM.

     https://audio.com/audius/audio/beaumont-rag-mandolin-130-bpm

I hope you can update it soon. Or if need be, coordinate so the Guitars Vol 2 mandolin and MuseScore Studio audio engine work properly together.

Looking forward to hearing back soon! And it would be great if an representative would respond at the Github discussion linked above.

**Comment by scorster** [comment]
## UPDATE:  I can't get work done with my purchased MuseHub libraries

1) It's been almost three months and I still haven't heard back from Muse Hub regarding sound quality inconsistencies in the mandolin in Guitars Vol.2. Is this the way it's supposed to work?

2) I also  initiated contact with MuseHub via their support mechanism [regarding Muse Guitars Vol 2 disappearing from MuseScore's Mixer menu](https://musescore.org/en/node/372335#comment-1283873). This was about the same time as my followup "sound quality" inquiry (posted directly above.) 

A couple of days later I received a generic message from MuseHub with links to various MuseScore.org and Github forums. It read like a boilerplate response, suggesting only that I seek help by posting here on Github and on .org, which as you can see I had already done.

I wrote back encouraging MuseHub itself to detail a solution for my particular MuseHub/MuseSounds situation. A few days later, after receiving no further reply, I suggested they respond here.

... crickets ... 

I guess MuseHub is relying on Github and .org to assist users. But the only forum/community response I received was from a person with similar problems who was also seeking a solution. And as of yet there's no input from the Github development community.   

For the record, my MacOS system library shows a number of MuseSounds downloaded, including

~/Library/Application Support/Muse Hub/Downloads/Instruments/**Muse Guitars Vol 2**     << Dated: 2024-12-29

**Comment by scorster** [comment]
Tomorrow will be day 10 without any help from MuseHub, so I posted a negative review for Guitar Vol 2.

https://www.musehub.com/muse-sounds/muse-guitars-vol-2/reviews

**Reply by Dima-S-Jr** [reply]
> Tomorrow will be day 10 without any help from MuseHub, so I posted a negative review for Guitar Vol 2.
> 
> https://www.musehub.com/muse-sounds/muse-guitars-vol-2/reviews

And you're not the only one facing this kind of problem.
These guys are really annoying to ignore users. At least they would have answered something, but no, it's better to keep silent. They won't tell you why it's taking so long to fix the bugs. After all, they only have 2 people working on it (imagine - 2 people!) and most likely it's a second-rate (if not third-rate) job for them, for which they apparently get paid pennies.

My requests are generally closed with a "Solved" mark. You can get acquainted with some of them ([#4518](https://support.musehub.com/hc/en-gb/requests/4518), [#4034](https://support.musehub.com/hc/en-gb/requests/4034), [#4031](https://support.musehub.com/hc/en-gb/requests/4031), [#3923](https://support.musehub.com/hc/en-gb/requests/3923), [#4039](https://support.musehub.com/hc/en-gb/requests/4039), [#4580](https://support.musehub.com/hc/en-gb/requests/4580), etc.). And if they respond to paid libraries in this way, then imagine their level of indifference towards free libraries.

I think this way: if you get down to business, if people pay money for it, then do it with dignity, and not in such a way that problems appear later that you will cope with, perhaps, only in a few years. After all, there are such simple problems that can be solved pretty quickly. Why are they dragging their feet? Yes, the free HALion SO is much better than Muse, even though there is no mute support for brass and strings.

**Reply by scorster** [reply]
@Dima-S-Jr

Well said. Thanks for you comments! 

I'd like to see your unresolved cases, but maybe they're private? I get "oops The page you were looking for doesn't exist." Then I thought I needed to be logged in. And though musehub.zendesk.com is frequently problematic during attempts to login, one of my identities worked. But still I encounter the same "page doesn't exist" alert.

And apparently the MuseHub forum is just another potentially pointless place to post questions. Here's a post of mine from just over a year ago:

https://support.musehub.com/hc/en-gb/community/posts/16629853036445-Where-is-the-MuseFX-documentation

What a mess

**Reply by Dima-S-Jr** [reply]
![Снимок экрана (923)](https://github.com/user-attachments/assets/f36f293c-12e2-4832-b7ba-2376a8cd307d)
![Снимок экрана (922)](https://github.com/user-attachments/assets/d7de7223-6497-4251-a682-a955efc0894a)
![Снимок экрана (921)](https://github.com/user-attachments/assets/9e1f5d07-ab10-4ac4-ad6a-811338c380da)
![Снимок экрана (920)](https://github.com/user-attachments/assets/4104f084-c1b3-4786-9480-fc0b35dbe0ae)
![Снимок экрана (919)](https://github.com/user-attachments/assets/d7b5064b-6234-487c-bcc9-887b3828b4e5)
![Снимок экрана (918)](https://github.com/user-attachments/assets/b6782543-0c8e-441e-8d71-6d0ac4eab1a0)

**Reply by scorster** [reply]
Thanks for posting a short history of your MuseHub support ticket. 

Sad story.

**Reply by scorster** [reply]
> > Tomorrow will be day 10 without any help from MuseHub, so I posted a negative review for Guitar Vol 2.
> > https://www.musehub.com/muse-sounds/muse-guitars-vol-2/reviews

@Dima-S-Jr wrote:

> And you're not the only one facing this kind of problem. These guys are really annoying to ignore users. At least they would have answered something, but no, it's better to keep silent. They won't tell you why it's taking so long to fix the bugs. After all, they only have 2 people working on it (imagine - 2 people!) and most likely it's a second-rate (if not third-rate) job for them, for which they apparently get paid pennies.

> I think this way: if you get down to business, if people pay money for it, then do it with dignity, and not in such a way that problems appear later that you will cope with, perhaps, only in a few years. After all, there are such simple problems that can be solved pretty quickly. Why are they dragging their feet? Yes, the free HALion SO is much better than Muse, even though there is no mute support for brass and strings.

Sadly, it's nearly a year since I initially post this topic here on Github ... and there's been no response from MuseScore development either.

**Comment by scorster** [comment]
In various organisms an organ can compensate for a deficiency in another organ.

I've been hoping that within MuseGroup the various organs (Github and MuseScore.com) would compensate for MuseHub's lack of support. These Github discussions date back 3 months and 2 weeks, but I'm still hoping for a reply and a solution or some sense of progress.

For some reason (as stated above) MuseHub seems to thinks that's how it should work ...

**Comment by scorster** [comment]
I'm now wondering if the problem could emanate from an incomplete or inaccurate json file.

Should my user Application Support folder house a json file that contains a reference to my purchase of Muse Guitars Vol 2? As reported above the Guitars Vol 2 library files appear to be where they belong.

**Comment by scorster** [comment]
It's 12 days since I submitted an issue support request from MuseHub.

I now reply everyday to support@musehub.org (Aydar Faretdinov) the replyTo address I received in the initial generic "response." This is today's message:

*Day 12 since filing an issue report. To date, no sentient reply.*

*I'm hoping someone will step up to the plate regarding MuseGuitars Vol 2 not showing in MuseScore Studios Mixer sound menu. Also I'm still hopeful for a reply regarding playback issues with onsets, durations, volume and pitch with the Mandolin in that library.*

**Comment by scorster** [comment]
## Day 14

Today bare-handedly I fixed my stuck point with MuseScore's Studio Mixer>Sound menu—where purchased MuseHub/MuseSounds libraries were failing to display, which left me unable to use them.

The following steps may help if you're in a similar situation. Essentially they amount to Uninstall/Download (within the MuseHub desktop app):

     1.  close MuseScore Studio
     2.  open the MuseHub app (NOTE: Logging into MuseHub.org does not appear to be a viable course.)
     3.  if needed, log in through the app, which seems to always require a verification code
     4.  click on the Library "tab" in the bottom left
     5.  locate a problematic/missing library (for me, Muse Guitars Vol 2) 
     6.  chose **Uninstall** from the "down arrow" menu in the right column
     7.  when the uninstall finishes a Download button appears
     8.  click the Download button and wait for the download to complete
     9.  close MuseHub
     10.  open MuseScore Studio
     11.  After opening a score that used Muse Guitars Vol. 2, go the Mixer and re-select the instruments you have previously assigned.

**Result:** Best I can tell **ALL my missing libraries now appear in Mixer>Sound>MuseSounds>Muse**

Over the last two weeks I've exerted  persistence with my support ticket, but MuseHub didn't help in the least.

This image shows my purchased **Guitar Vol 2 library** (In MuseHub desktop app) The popup menu pertains to **Guitar Vol 2 library**

![MuseHub Guitars Vol 2 FIX - Uninstall via the MuseHub app, then reinstalled](https://github.com/user-attachments/assets/4fc08d63-7f24-44ee-90b3-3d83323c1832)

Here's an image of the cloud/Download button:

     ![MuseHub - Cloud Download button](https://github.com/user-attachments/assets/85a7d759-2e27-4a64-900f-be5ee796d474)

**Reply by scorster** [reply]
This Muse Sounds issue arose again today, out of the blue, after months Guitars Vol. 2 being accessible in the Mixer, today I found it MIA. Just as reported above. Although there were a couple of login issues, and the following alert appeared, even though I was currently logged into .org:

![MuseHub Login Issue](https://github.com/user-attachments/assets/55b649a2-95b5-4aa2-9106-26428a2ed661)

Though it cost me about 20 minutes of my time, fortunately, the steps I listed above again solved the problem I keep encountering with MuseHub and/or MuseSounds, which is of fragile installation integrity.

**Reply by scorster** [reply]
My installation of Muse Guitar Vol. 2 disappeared again today. And presently the trigger that continually causes this is unknown. 

The unfortunate result: All tracks that were assigned to a Muse Guitars Vol. 2 instrument revert to MSBasic.

Today it only took 10 minutes to reinstall Muse Guitars Vol. 2 and Muse Strings. That's because I referred to the steps I outlined above that remedy the situation.

**Comment by scorster** [comment]
The MuseSounds post-installation MIA issue has been reported by others:

https://musescore.org/en/node/386098

https://musescore.org/en/node/385023

**Reply by scorster** [reply]
The **Guitars Volume 2** became inoperable again on my system months ago. I decided to let it go until using the initial release of MSS 4.7. I installed 4.7 yesterday and [followed my remedy steps (above)](https://github.com/orgs/musescore/discussions/25963#discussioncomment-12665334). Doing so fixed the situation for the nth time. Hopefully it will stick now:

 - MSS 4.7.0-260513133, revision: github-musescore-musescore-9989a50
 -  MacOS 26(Tahoe)
 - MuseSampler 0.105.1

Disappointing that this disabling bug persists when I reported here and to MuseHub at least 1.25 years ago.

=======================
BTW, There's a font fallback issue. I've mainly used the Mandolin that comes with **Guitars Vol. 2.** During a Vol. 2 "blackout" (when I open and edit a score that used **Guitars Volume 2's** mandolin) the score falls back on the MSBasic Mandolin. BUT when I reinstall **Guitars Vol. 2.** the score ONLY remembers to use MSBasic's Mandolin—i.e. it doesn't keep record of the intended font, but only the Fallback font. I would expect that when **Guitars Vol. 2** is back "online" MSS4.7 would know to use IT rather than the MSBasic fallback mandolin.

Should I create a seperate Github post on this shortcoming?

**Comment by scorster** [comment]
System specs:

- MacOS 26.5
- MuseScore Studio 4.7.4
- MuseHub 3.2.1(4226)
- MacBook Air M2

Months ago the **Muse Guitars Vol. 2 library** went missing yet again, however I put off reinstalling because I rarely use MuseScore Studio 4.7.4.

Today I resolved the issue again using the steps I outlined above. But that does resolve fundamental issue and question: Why does the MuseScore keep losing its reference to MuseScounds: **Guitars Vol. 2**. THAT would be the issue to fix. And it's worth noting that the **Guitars Vol. 2** files are in the correct installation location. There just seems to be some MuseScore/MuseSounds failure to reference them.

MuseHub support has been of zero assistance in this recurring, 1.5 year-long issue.

Maybe the MuseScore development should step up. Seems like it would matter that MuseHub, MuseSounds and MuseHub work smoothly together, that references or pointers to sound libraries persist, or be easily reestablished via MuseScore—because, despite Guitar Vol. 2 not appearing in MuseScore Studio, the Guitar Vol. 2 library has always been resident on my MacBook.

**Comment by scorster** [comment]
Anyone from the MuseScore side want to take this up? Investigate? Collaborate with MuseHub? 

I'd think the two arms of Muse would want to join forces to resolve well documented issues.
